query-engine/driver-adapters/executor/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "engines": {
-    "node": ">=16.13",
-    "pnpm": ">=8.6.6 <9"
+    "node": ">=18.18",
+    "pnpm": ">=8.6.6"
```

Comment:
I think this should be limited to a specific major version because of different lockfile versions, if we are going to commit the lockfile. If you used pnpm 9 locally then let's just specify pnpm 9 and that's it.

Reply:
Right, let's fix this in a follow-up PR.